tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+# role tasks
+- name: Upgrade kexec-tools to latest
+  yum:
+    name: kexec-tools
+    state: latest
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Waive previous nfs setup if enabled
+  lineinfile: dest="{{ kdump_conf_path }}"
+              state=present
+              regexp='^nfs (.*)$'
+              line='#nfs \1'
+              backup=true
+              backrefs=true
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Set nfs kdump target for crash kernel
+  lineinfile: dest="{{ kdump_conf_path }}"
+              state=present
+              regexp='^(#?)dracut_args (.*)$'
+              line='dracut_args --mount "kdump.usersys.redhat.com:/var/www/html/vmcore /mnt/vmcore nfs defaults"'
+              backup=true
+  become: true
+  become_user: "{{ kdump_become_user }}"
+
+- name: Set relative path of vmcore folder
+  lineinfile: dest="{{ kdump_conf_path }}"
+              state=present
+              regexp='^path (.*)$'
+              line='path /{{inventory_hostname}}'
+              backup=true
+  become: true
+  become_user: "{{ kdump_become_user }}"
+  register: kdump_conf
+
+- name: Enable kdump service
+  systemd:
```

Comment:
You don't have a `state: started` here. Is that intentional?

Additionally, let's switch to using the `service` module where possible. I understand it doesn't have the `masked` option, but I'm not sure that's necessary. If it is, then let's push the `enabled: true` down to a following task and use `service` there.